lib/resources.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'cgi'
 require 'yajl/json_gem'
+require 'pry'
```

Comment:
I think we should remove 'pry'

Reply:
good catch updated [here](https://github.com/cerner/fhir.cerner.com/pull/608/commits/6c66ea69273dd5e28815440012b55df4d71ee2df)